### DISCUSSION TITLE
UI/UX bug

### DISCUSSION THREAD

**Discussion by ptchela** [subquery/subql#945] [Frontier TestNet - Bug Reporting]
![4545](https://user-images.githubusercontent.com/80599907/165717494-377a33d7-07c9-4175-8d97-e294398e45c0.jpg)

The [Explorer](https://frontier.subquery.network/explorer) interface is buggy, the correct values ​​are not displayed. For example, this project is fully indexed, it should be 100% and full scale.

Also in the node form the "not indexing" flag often hangs, although in fact projects are indexed.

**Comment by ianhe8x** [comment]
Good catch.

**Comment by lilian79511** [comment]
Ticket created: [SQN-420]
